CMakeLists.txt

```diff
@@ -26,6 +26,16 @@ include(GNUInstallDirs)
 include(FortranCInterface)
 include(CMakePackageConfigHelpers)
 
+# Requires OpenMP
```

Comment:
Thanks, that makes sense. Do you happen to know if I should be using `recursive` for every Fortran lib (e.g., gdma, dkh, pcmsolver, v2rdm_casscf) that links into Psi4?

And do you happen to know if I should be forcing `ifcoremt` instead of `ifcore`, as [this suggests (search ifcore)](https://software.intel.com/en-us/articles/threading-fortran-applications-for-parallel-performance-on-multi-core-systems), esp. for static linking?

```
-rwxr-xr-x. 1 root root 2133490 Apr 15  2016 /intel2016/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64/libifcore.a
-rwxr-xr-x. 1 root root 2181924 Apr 15  2016 /intel2016/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64/libifcoremt.a
-rwxr-xr-x. 1 root root 1872780 Apr 15  2016 /intel2016/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64/libifcoremt_pic.a
-rwxr-xr-x. 1 root root      26 Apr 15  2016 /intel2016/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64/libifcoremt.so
-rwxr-xr-x. 1 root root 1438433 Apr 15  2016 /intel2016/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64/libifcoremt.so.5
-rwxr-xr-x. 1 root root 1840986 Apr 15  2016 /intel2016/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64/libifcore_pic.a
-rwxr-xr-x. 1 root root      24 Apr 15  2016 /intel2016/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64/libifcore.so
-rwxr-xr-x. 1 root root 1421112 Apr 15  2016 /intel2016/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64/libifcore.so.5
```